table/tables/tables.go

```diff
@@ -491,6 +491,10 @@ func (t *tableCommon) AddRecord(ctx sessionctx.Context, r []types.Datum, opts ..
 			colIDs = append(colIDs, col.ID)
 			row = append(row, value)
 		}
+		if !t.canSkipInsertBinlog(col, value) {
```

Comment:
can we update the function `canSkip`?

Reply:
if use the same skip func, this will change the data write to KV too, writing column value event the value is null and default value is null, i think currently it only void to write this column at such case to save space. @crazycs520 PTAL